### ISSUE TITLE
PowerShell ISE Snippets not following the new style

### ISSUE THREAD

**Issue by DarkLite1** [pester/Pester#982]
When pressing CTRL + J in the PowerShell ISE we can see the Pester snippets. They are currently implemented without the dash:

| Should BeGreaterThan 

It should be

| Should -BeGreaterThan 

This is for all snippets from Pester the case.

**Comment by it-praktyk** [comment]
IMHO, PowerShell ISE Snippets should be excluded from the main Pester repository.

@DarkLite1, we are open for your contribution. Please update not only report. Thank you.

**Comment by DarkLite1** [comment]
You are correct, when I have some time I'll try to set up Git and do a commit.
